### PR TITLE
MAX! Fix initiation issue

### PR DESCRIPTION
**Pull request by marcelrv**

### CODE REVIEW

File: addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java [hunk 1]
```diff
@@ -555,6 +556,22 @@ private void setProperties(Device device) {
      *
      * @param device
      */
+    /* (non-Javadoc)
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: this second JavaDoc seems superfluous.

File: addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java [hunk 2]
```diff
@@ -555,6 +556,22 @@ private void setProperties(Device device) {
...
+     * @see org.eclipse.smarthome.core.thing.binding.BaseThingHandler#bridgeStatusChanged(org.eclipse.smarthome.core.thing.ThingStatusInfo)
+     */
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        logger.debug("Bridge Status updated to {} for device: {}", bridgeStatusInfo.getStatus().toString(),
+                getThing().getUID().toString());
+        getMaxCubeBridgeHandler();
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: Is this line necessary? The handler should always be there and still be the same as during initialize() is called.

Comment (marcelrv) [inline addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java]: fair. I removed it

**Comment by kaikreuzer** [comment]
thx!

**Pull request merged**